common/server/http-stats-t.h

```diff
@@ -0,0 +1,78 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2025 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include "common/stats/provider.h"
+
+class http_stats_t final : public stats_t {
```

Comment:
Let's make not confusing naming and name it `open_metric` in all places
```suggestion
class open_metrics_stats_t final : public stats_t {
```